.github/workflows/e2e_android.yml

```diff
@@ -7,7 +7,7 @@ name: '[e2e] Android Detox Test Run'
 on:
   push:
     branches:
-      - 'dependabot/**'
+      - develop
```

Comment:
is the objective to run the detox tests after it is pushed to develop? That is what this line would do

Reply:
if that is the objective here is to run after a pr is merged then I think this is good to go

Reply:
The objective is to have it run whenever someone makes a change to their PR.  I know we talked about it (and I forgot to take the code out) so I think its correct now but correct me if I'm still wrong.